### PR TITLE
Method router configuration

### PR DESCRIPTION
**Pull request by babky-atteq**
Just a refactor to the way how the methods are configured

### CODE REVIEW

File: zsl/application/modules/web/configuration.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from zsl.configuration import InvalidConfigurationException
```
Comment (morihladko) [inline zsl/application/modules/web/configuration.py]: Put
```python
from __future__ import (absolute_import, division,
                        print_function, unicode_literals)
from builtins import *
```
at the beginning to be python2 compatible.

File: zsl/configuration/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class InvalidConfigurationException(Exception):
```
Comment (morihladko) [inline zsl/configuration/__init__.py]: Add `from __future__ ..` imports.

File: zsl/interface/webservice/utils/request_data.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+def extract_data(request):
```
Comment (morihladko) [inline zsl/interface/webservice/utils/request_data.py]: Again, you forgot to add the py2/3 compatible imports.

File: zsl/application/modules/web/configuration.py (hunk 1 above)
Comment (babky-atteq) [inline zsl/application/modules/web/configuration.py]: done

File: zsl/interface/webservice/utils/request_data.py (hunk 3 above)
Comment (babky-atteq) [inline zsl/interface/webservice/utils/request_data.py]: done

File: zsl/configuration/__init__.py (hunk 2 above)
Comment (babky-atteq) [inline zsl/configuration/__init__.py]: already done

**Pull request merged**